otherlibs/runtime_events/runtime_events_consumer.c

```diff
@@ -943,6 +940,7 @@ static int ml_user_unit(int domain_id, void *callback_data, int64_t timestamp,
 
     // payload is prepared, we call the callbacks sequentially.
     if (user_events_call_callback_list(holder, callback_list, params) == 0) {
+      CAMLdrop;
       return 0;
```

Comment:
This is not wrong technically, but at first I wanted to suggest using the documented macro `CAMLreturnT(int, 0);`. Idem in other places below. On second thought, I do not have much objective arguments against using `CAMLdrop` in this way, so it is a matter of style mainly. So I am ok with this, if people agree that `CAMLdrop` is now in common use.

Reply:
`CAMLdrop` is not in common use, it looks like the only file making a non-exceptional use of it is this one, and the present PR might be taken as an indication that it is error-prone and should be avoided.

Reply:
Indeed, "don't use `return` in conjunction with `CAMLparam`" is a good rule of thumb.

Reply:
Thanks, I'll replace the uses of `CAMLdrop`.